packages/notifi-react-card/lib/components/subscription/EventTypePriceChangeRow.tsx

```diff
@@ -69,16 +123,17 @@ export const EventTypePriceChangeRow: React.FC<
 
     if (!enabled) {
       setEnabled(true);
-      instantSubscribe({
-        alertConfiguration: alertConfiguration,
-        alertName: alertName,
+      subscribeAlert({
+        eventType: config,
+        inputs: {},
```

Comment:
is it okay that `inputs` is always an empty object?

Reply:
in [`ensurePriceChangeSource`](https://github.com/notifi-network/notifi-sdk-ts/blob/b6881792f87ffecb8713ce83aa4a23eaecb2853f/packages/notifi-frontend-client/lib/client/ensureSource.ts#L113), it does not make use use of `inputs` to `PriceChangeEventTypeItem`. So the inputs arg is always `{}`.